soccer/src/soccer/strategy/coach/coach_node.cpp

```diff
@@ -76,6 +81,10 @@ void CoachNode::ball_sense_callback(const rj_msgs::msg::RobotStatus::SharedPtr m
 }
 
 void CoachNode::check_for_play_state_change() {
```

Comment:
change the name of this so it's less confusing 

should be a top-level "coach node tick" that calls "check for play state" and also "assign positions" instead of putting assign in here